tests/test_turbulence.py

```diff
@@ -0,0 +1,112 @@
+"""Tests of ParameterGenerator() methods involving turbulence parameters."""
+import psfws
+import numpy as np
+
+
+def test_joint_pdf():
+    # iterate through some semi-random values of rho
+    rho_goal = [0.2, 0.45, 0.62, 0.83]
+    rho_result = []
+    for rho_jv in rho_goal:
+        p = psfws.ParameterGenerator(rho_jv=rho_jv, seed=23459)
+        rho_result.append(np.corrcoef(p.data_gl['j_gl'],
+                                      p.data_gl['speed'])[0][1])
+    # check whether results are within some tolerance of desired values
+    np.testing.assert_allclose(rho_result, rho_goal, rtol=.1, atol=.05,
+                               err_msg='correlating marginal pdfs failed.')
+
+
+def test_turbulence_draws():
+    # first test with no wind correlation for ground layer
+    j_gl_goal = 3.880497137403186e-13
+    j_fa_goal = 1.936532096294314e-13
+    p = psfws.ParameterGenerator(rho_jv=0, seed=460)
+    j_fa_res, j_gl_res = p._draw_j()
+
+    np.testing.assert_equal([j_fa_res, j_gl_res], [j_fa_goal, j_gl_goal],
```

Comment:
Probably want all_close here.  Numerical calculations like this can end up slightly different from one system to the next at the 16th decimal place.  Testing all_close at rtol=1.e-12 is generally good for things that you expect to be basically exactly the same, but might have numerical variation like that.